service/controller/resource/tccpn/template/params_main_security_groups.go

```diff
@@ -0,0 +1,19 @@
+package template
+
+type ParamsMainSecurityGroups struct {
+	APIWhitelistEnabled        bool
+	PrivateAPIWhitelistEnabled bool
```

Comment:
Adhering to the alphabet would be fantastic. 🤗

Reply:
Pick your consistency.

https://github.com/giantswarm/aws-operator/blob/master/service/controller/resource/tccp/template/params_main_security_groups.go#L7

Reply:
Not sure what you mean. When this was already "wrong" in another place, let's just make it better now.